documentation/learnmore/releaseNotes.html

```diff
@@ -33,7 +36,7 @@ <h3>Database compatibility</h3>
     <h3>New features</h3>
     <ul>
         {% include issue.html id="2921" title="Add Log4J v2 support" %}
-        {% include issue.html id="2919" title="Allow custom S3Client to be supplied" %}
+        {% include issue.html id="2919" title="Allow custom S3Client to be supplied (reverted in next release)" %} 
```

Comment:
This isn't strictly true is it? A custom client can still be supplied, just not via `FluentConfiguration`